File: NET/JNetCall.Sharp.Tests/MainCallTest.cs
// ReSharper disable UnusedMember.Global
using JNetCall.Sharp.Client;
using static JNetCall.Sharp.Client.Tools.ServiceEnv;

namespace JNetCall.Sharp.Tests
{
    public sealed class MainCallTest : CallTest
    {
        internal readonly string Path
            = BuildPath(@"..\..\JVM\alien1-java\target\alien1-java.jar");

        protected override T Create<T>()
        {
            return StdIOClient.Create<T>(Path);
        }
    }
}

File: JVM/jnetcall-java-tests/src/test/java/jnetcall/java/tests/MainCallTest.java
package jnetcall.java.tests;

import static jnetcall.java.client.tools.ServiceEnv.buildPath;

import jnetcall.java.client.StdIOClient;

public final class MainCallTest extends CallTest {

    final String Path =
        buildPath("..\\..\\..\\NET\\Alien1.Sharp\\bin\\Debug\\net6.0\\Alien1.Sharp.exe");

    @Override
    protected <T extends AutoCloseable> T create(Class<T> clazz) {
        return StdIOClient.create(clazz, Path);
    }
}
